tests/system/providers/amazon/aws/example_rds_instance.py

```diff
@@ -71,6 +73,22 @@
     )
     # [END howto_sensor_rds_instance]
 
+    # [START howto_operator_rds_stop_db]
+    stop_db_instance = RdsStopDbOperator(
+        task_id="stop_db_instance",
+        db_identifier=rds_db_identifier,
+        wait_for_completion=True,
```

Comment:
`wait_for_completion=True` is a default value, you can drop it from this call (similar to how RdsCreateDbInstanceOperator` does above.